tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -820,8 +820,17 @@ def test_parse_search_registered_models_order_by(self):
         self.assertEqual([str(x) for x in parsed], ['registered_models.name DESC'])
 
         # test that an exception is thrown when order_by contains duplicate fields
-        with self.assertRaisesRegex(MlflowException, 'order_by contains duplicate fields:'):
-            SqlAlchemyStore._parse_search_registered_models_order_by(['name', 'name'])
+        order_bys = [
```

Comment:
If you want, you can split this out into a separate test and use `pytest.mark.parametrize`

Reply:
Unfortunately, pytest parametrization does not work in unittest.TestCase subclass

https://docs.pytest.org/en/stable/unittest.html#pytest-features-in-unittest-testcase-subclasses